### DISCUSSION TITLE
Can we delete the first empty row in `flows-partition.csv`?

### DISCUSSION THREAD

**Discussion by gnawin** [TulipaEnergy/TulipaEnergyModel.jl#518] [Questions] [answered]
The first row in `flows-partition.csv` is empty.

It can create trouble when you edit the csv using excel. My user case is when I delete everything starting from row 3, i.e., when I don't want any partitions, save it, then the first row will be gone. This will give an error when doing IO.

While it may not be good practice to edit csv using excel, it is by far the most convenient way to manipulate data, and thus we should make this less error-prone. 

So, can we delete the first empty row in `flows-partition.csv`?

**Comment by datejada** [comment, marked as answer]
I would prefer to have some meaningful names or options in those files for the first row like this:

```
,,{uniform;explicit;math},
asset,rep_period_id,specification,partition
```

Then you won't have a problem opening in Excel, and we can keep consistency in the code such that our data always starts in the third row.
